### ISSUE TITLE
[Improvement] Account - Opportunities (relation tab/widget)

### ISSUE THREAD

**Issue by nikdejan** [YetiForceCompany/YetiForceCRM#2501]
Should I say it... like we used to have dedicated Opportunities widget for Accounts module

**Comment by nikdejan** [comment]
And, off course, we also need Sales Processes tab in Accounts.

**Comment by bpabiszczak** [comment]
Soon we'll add it.

**Comment by nikdejan** [comment]
Today it is "soon" :-)
Please add related Opportunities tab to Accounts module. It's is really fundamental.
Opportunities - Accounts is the most important relation in the whole CRM.

**Comment by nikdejan** [comment]
What does YF list of priorities look like if you don't implement such a fundamental relation for a such a long time? Unless there is some hidden strategy, I suppose you are simply missing the point of this issue. So I changed issue title to be more descriptive.

**Comment by bpabiszczak** [comment]
Looking at it only from the perspective of your own needs doesn't mean that it's beneficial for the project itself. Something that seems extremely important to you might be of low priority for us. Pressure doesn't have any effect on me, neither does the stronger impact you're trying to achieve. It doesn't matter to me if you write 1 or 101 posts. You can't see the changes that we introduce at the same time and you think your problem is the most important one.. well, it isnt...

“Whoever intends to have the right, if but his tongue be clever,
Will have it, certainly.”
- Johan Wolfgang von Goethe

**Comment by waran70** [comment]
So why nikdejan proposed  Account - Opportunities (relation tab/widget) is
not logical/useful from the point of view of sales/marketing process?
Can you list cons?




On Thu, Jan 21, 2016 at 5:27 PM, Błażej Pabiszczak <notifications@github.com>
wrote:
>
> Looking at it only from the perspective of your own needs doesn't mean
that it's beneficial for the project itself. Something that seems extremely
important to you might be of low priority for us. Pressure doesn't have any
effect on me, neither does the stronger impact you're trying to achieve. It
doesn't matter to me if you write 1 or 101 posts. You can't see the changes
that we introduce at the same time and you think your problem is the most
important one.. well, it isnt...

**Comment by nikdejan** [comment]
@bpabiszczak:  "Looking at it only from the perspective of your own needs doesn't mean that it's beneficial for the project itself. Something that seems extremely important to you might be of low priority for us."
There are many things that I need but still don't ask for them, because I know that they are specific and that majority of people won't need them. So I don't expect you, or community or anyone else to adapt to my needs. 
BUT, at this particular issue I am asking for something that is so fundamental that everyone needs it! Not only that everyone needs it but every known CRM has it! Vtiger has it! Sugar CRM has it! Suite CRM has it! Yetiforce 2.3 has it! I am certain that YetiForce 0.01 had it!
I never meant to make pressure on you, and I'm not repeating my request 101 times. 
I requested this over a month ago and after that you responded: "Soon we'll add it". (So, at this point you made me think that it IS a priority to you). And you immediately closed the issue, which is wrong, because you should only close an issue when you somehow resolve it, and not when you PROMISE to resolve it. You might never thought about it that way, but issue tracking is not only meant for you. It is also useful for us, because we, the community, want to track the development so when some critical issues (critical for us - yeah we are allowed to think about our own interest) are resolved we might start using yetiforce. 
When you close an issue you are moving it off the table, and that is why I wanted to remind you that the issue is still there. And it seams to me that you thought I was right at that moment, because you reopened the issue.
Almost two weeks later I wrote a comment in which I expressed wonder that this fundamental functionality is not implemented yet. And what I said I really meant, without any sarcasm: "Unless there is some hidden strategy, I suppose you are simply missing the point of this issue. So I changed issue title to be more descriptive." Mystery remains until our days. 
Maybe I don't see everything you do, but I do see a lot of things. And there are many good things. And there are many things that are useless or, at least, not important but I guess someone is paying you to implement them and I'm OK with that. Aren't we all doing the same? And there are over 60 changes you implemented after my bug report or feature request. I have been trying to be constructive with issues I opened and this has been recognized by your team by implementing the changes I requested. I have been trying to be constructive by answering some beginner questions (as much as I could) and leave you more time for advanced work and you recognized that by politely saying "thank you" each time.
It's not good idea to use harsh language to people who are basically constructive, although sometimes we might think differently. 
I strongly believe that YF has a chances of becoming best open crm in the world. One good thing about that could be that you could hire a PR that knows English. Good PR will avoid misunderstandings, reduce unnecessary and exhausting quarrels about ethical issues and she will reduce the need to translate all this quarrels.  :-)

**Comment by bpabiszczak** [comment]
@waran70 – I said it was of low priority for us, not that it was useless or pointless to add.

@nikdejan – you take it too personally, the most important information is that i said we would eventually add it to the system. I closed the issue, so it wouldn't just sit there for no reason. This functionality is - as you said - fundamental and we do know it must be added. Just like in Opportunity there's no STATUS/MILESTONE field, and this field (as well as a bunch of other things) will be added before 2.4 release, because they're obvious. However, some elements have to be added in certain order [why would anybody need a widget if there's no basic info like milestone for example].

We've got two girls who translate... but sometimes when I'm in hurry I post using translators... Although majority of the posts [90%] is verified by them.
